plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlClient.java

```diff
@@ -14,7 +14,7 @@
 package io.trino.plugin.mysql;
```

Comment:
Can you please also take a look at MySqlClientModule and remove and connection properties that are now obsolete. I do remember `nullCatalogMeansCurrent` defaults to `false` now so can be removed.